defaults/main.yml

```diff
@@ -15,8 +15,5 @@ duounix_login_config:
   pushinfo: "yes"
   autopush: "no"
   prompts: 3
-duounix_pam_config:
-  failmode: secure
-  pushinfo: "yes"
-  autopush: "no"
-  prompts: 3
+duounix_pam_config: "{{ duounix_login_config }}"
```

Comment:
Technically a breaking change since a user may have only overridden the `duounix_login_config` variable.

Probably not a big deal, though.